tests/expected/shadow/unsupported_num_objects/test.rs

```diff
@@ -32,10 +32,10 @@ fn check_max_objects<const N: usize>() {
 
 #[kani::proof]
 fn check_max_objects_pass() {
-    check_max_objects::<1021>();
+    check_max_objects::<510>();
```

Comment:
Quite a drop here.

Reply:
Ack, but not one related to performance: it's just a different way of interpreting what is/is not an object.